src/logger.js

```diff
@@ -158,7 +159,11 @@ class Logger extends Writable {
       } else {
         connection = net.connect.apply(null, args);
       }
-      connection.setTimeout(opts.inactivityTimeout || defaults.inactivityTimeout);
+      if(opts.disableTimeout){
+        connection.setTimeout(0);
```

Comment:
Changed it up. Also noticed that I wasn't setting a default value for disableTimeout so fixed that up.